Sources/SWBUtil/HashContext.swift

```diff
@@ -173,25 +364,29 @@ public class DelegatedHashContext: HashContext {
     }
 }
 
-public final class MD5Context: DelegatedHashContext {
+public final class InsecureHashContext: DelegatedHashContext {
     public init() {
         #if os(Windows)
         super.init(impl: BCryptHashContext(algorithm: "MD5", digestLength: 16))
-        #else
+        #elseif canImport(CryptoKit)
         super.init(impl: SwiftCryptoHashContext<Insecure.MD5>())
+        #else
+        super.init(impl: VendoredSHA256HashContext())
```

Comment:
This seems a little shaky to use MD5 on Windows and Apple but SHA256 on other platforms. Do you plan to unify this further? I'm even surprised the tests pass like this.

Reply:
I think we should probably standardize on SHA256 everywhere in the future. Very few tests actually depend on the specific hash values, it's just important that they remain stable across builds performed by the same build system executable